sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -26,11 +26,69 @@ import { WebSocketImpl } from 'rhea-promise';
 
 export { AccessToken }
 
+// @public
+export interface AmqpAnnotatedMessage {
+    applicationProperties?: {
+        [key: string]: any;
+    };
+    body: any;
+    deliveryAnnotations?: {
+        [key: string]: any;
+    };
+    footer?: {
+        [key: string]: any;
+    };
+    header?: AmqpMessageHeader;
+    messageAnnotations?: {
+        [key: string]: any;
+    };
+    properties?: AmqpMessageProperties;
+}
+
+// @public
+export const AmqpAnnotatedMessage: {
+    fromRheaAmqpMessage(msg: AmqpMessage): AmqpAnnotatedMessage;
+};
+
 export { AmqpMessage }
 
-export { AmqpMessageHeader }
+// @public
+export interface AmqpMessageHeader {
+    deliveryCount?: number;
+    durable?: boolean;
+    firstAcquirer?: boolean;
+    priority?: number;
+    timeToLive?: number;
+}
+
+// @public
+export const AmqpMessageHeader: {
+    toRheaAmqpMessageHeader(props: AmqpMessageHeader): MessageHeader;
+    fromRheaAmqpMessageHeader(props: MessageHeader): AmqpMessageHeader;
```

Comment:
Rhea would always meant that it is AMQP related, so consider changing `RheaAmqp` here to `Rhea`.
Also, do we really need to export these methods?
Same for the from/to in AmqpMessageProperties

Reply:
Done